### ISSUE TITLE
Support for time-enabled web maps?

### ISSUE THREAD

**Issue by karmstro77** [Esri/storymap-journal#22]
Will there be support for time-enabled web maps in the near future? One workaround would be to display a web page (instead of web map) and link to the time-aware map template but it would be nice to have native support for a time-enabled web map.

**Comment by glazou** [comment]
Yes, this is on our roadmap for later this year. The workaround you are suggesting is what we are recommending in the meantime.

**Comment by jeffbaranyi** [comment]
I'm glad to hear this is on the road map.  I agree with Kevin, this would make a nice addition.  

Here is the type of service I would like to include - http://tmservices1.esri.com/arcgis/rest/services/LiveFeeds/NDFD_SnowFall/MapServer

**Comment by oevans** [comment]
This is still being discussed, but in the meantime the time-aware template has been updated and works well inside Story Maps. See [this blog post](http://blogs.esri.com/esri/arcgis/2015/08/03/how-to-make-a-time-aware-story-map/) for more info.
